webpack.config.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+/* eslint-disable */ 
```

Comment:
- [ ] Kindly ensure you dont disable linter rules in your codebase because linters help to ensure that you write a bug-free, clean, and workable code. Always ensure you fix your linter errors instead of disabling them.

Reply:
- [ ] Please you forget to work on this, Kindly fix it